math/mode/mode.go

```diff
@@ -0,0 +1,40 @@
+// mode_test.go
+// author(s): [CalvinNJK] (https://github.com/CalvinNJK)
+// description: Finding Mode Value In an Array
+// see mode.go
+
+package mode
```

Comment:
This algorithm belongs in the `math` package not its own sub package.

Please move this to the `math` package.